homeassistant/components/growatt_server/sensor.py

```diff
@@ -105,6 +103,11 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     add_entities(entities, True)
 
 
+async def async_setup_entry(hass, config_entry, add_entities):
+    """Set up growatt server from Config Flow."""
+    hass.async_add_executor_job(setup_platform(hass, config_entry.data, add_entities))
```

Comment:
Don't call the function before passing it in.

```suggestion
    hass.async_add_executor_job(setup_platform, hass, config_entry.data, add_entities)
```